openwisp_radius/api/swagger.py

```diff
@@ -0,0 +1,55 @@
+"""
+The following serializers are not used to send
+response to user but in swagger documentation of
+the API.
+"""
+
+from rest_framework import serializers
+
+
+class ObtainTokenRequest(serializers.Serializer):
+    username = serializers.CharField(
+        max_length=150,
+        write_only=True,
+        help_text=('Username of the user for obtaining tokens.'),
+    )
+    password = serializers.CharField(
+        max_length=128,
+        write_only=True,
+        help_text=('Password of the user for obtaining tokens.'),
+    )
```

Comment:
this looks like a possible solution for https://github.com/openwisp/openwisp-users/issues/116, do you think it can be used there as well?
If yes, could you please add a reference in that issue?

Reply:
Yes, this can be used there, adding a reference.